iOS/Airbnb_iOS/Airbnb_iOS/Home/Controller/HomewViewController.swift

```diff
@@ -0,0 +1,127 @@
+//
+//  HomewViewController.swift
+//  Airbnb_iOS
+//
+//  Created by juntaek.oh on 2022/05/24.
+//
+
+import UIKit
+import CoreLocation
+
+class HomewViewController: UIViewController {
+
+    private let browseViewController = BrowseViewController()
+    private lazy var homeView = HomeView(frame: view.frame)
+    private let dataSource = SearchViewCollectionDataSource()
+    
+    private let locationManager = LocationManager()
+    private var currentLocation = CLLocation()
+
+    let searchBar: UISearchBar = {
+        let searcher = UISearchBar()
+        searcher.placeholder = "어디로 여행가세요?"
+        return searcher
+    }()
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        self.configureSettings()
+    }
+
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        self.navigationController?.hidesBarsOnSwipe = true
+    }
+}
+
+private extension HomewViewController {
+    
+    func configureSettings() {
+        self.addChild(browseViewController)
+        self.setHomeView()
+        self.setSearchBar()
+        self.setLocationAccess()
+    }
+    
+    func setHomeView() {
+        self.view = homeView
+        self.homeView.setDataSource(dataSource)
+    }
+    
+    func setSearchBar() {
+        searchBar.delegate = self
+        self.navigationItem.titleView = searchBar
+    }
+    
+    func setLocationAccess() {
+        self.locationManager.locationManager.delegate = self
+        self.alertLocationAccessNeeded(isDenied: self.locationManager.setLocationAccess())
+    }
+    
+    func alertLocationAccessNeeded(isDenied: LocationManager.AceessCase) {
+        var settingsAppURL: URL?
```

Comment:
기본값을 주고 아래처럼 else문 제거하는 방법이 있습니다🙂

```suggestion
        var settingsAppURL: URL? = URL(string: UIApplication.openSettingsURLString)
        
        if #available(iOS 15.4, *) {
            settingsAppURL = URL(string: UIApplicationOpenNotificationSettingsURLString)
        }
```